### DISCUSSION TITLE
Is it possible to put Timeshift on the Spiral Linux live ISO by default?

### DISCUSSION THREAD

**Discussion by savbran** [SpiralLinux/SpiralLinux-project#389] [General]
Hi, I ask the question in subject because it could be useful to use the same Spiral Linux Live ISO on a USB stick to restore through Timeshift the system in case that it won't start anymore.

What do you think? Is this a right question? Thansk in advance.

**Comment by geckolinux** [comment]
Hi there, I'm not familiar with the restore process that you mention with Timeshift, but it won't be included by default because it's incompatible with the Btrfs subvolume layout that Snapper needs. However you could install Timeshift or anything else you might need in the live USB stick session.

**Reply by DaAwesomeP** [reply]
It seems like Timeshift is similar to snapper but with a GUI? I know that you can restore Snapper partitions from a live disk. I wonder if there is another Snapper GUI that could work if Timeshift is not compatible.

**Reply by geckolinux** [reply]
They accomplish similar purposes but with major implementation and feature differences.

**Reply by DaAwesomeP** [reply]
Some other GUI options for BTRFS/Snapper that might work (I have not tested these):

- https://gitlab.com/btrfs-assistant/btrfs-assistant
- https://github.com/ricardomv/snapper-gui

@savbran You could possibly install one of these in the live ISO after booting it and it may work.

I have used the GUI in YaST before (definitely very solid), but obviously you don't get that on Debian.

**Reply by geckolinux** [reply]
@DaAwesomeP and @savbran  https://github.com/ricardomv/snapper-gui is preinstalled in SpiralLinux from the Debian repositories (https://packages.debian.org/bookworm/snapper-gui) .

**Reply by savbran** [reply]
Thanks guys for reporting some info here. Anyway I was using ext4 on my system but checking better how Btrfs works, perhaps I shoud use it instead of ext4.

**Comment by savbran** [comment]
I'm referring to the use case reported in [this forum post](https://forums.linuxmint.com/viewtopic.php?t=292382) in which the user wants to restore his system, not bootable anymore, launching Timeshift from the Live USB and point it to the snapshot that needs to be restored on the system.
